ts/components/messages/MessageDetailsInfoComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+
+import { format } from "date-fns";
+import enLocale from "date-fns/locale/en";
+import itLocale from "date-fns/locale/it";
+import _ from "lodash";
```

Comment:
import only the mudule(s) you use

Reply:
lodash  is used for capitalize function (row 38 & 40)

Reply:
i mean to load only the capitalize module and not the whole lodash